internal/devbox/docgen/readme.tmpl

```diff
@@ -7,11 +7,29 @@
 {{- if .Description }}
 {{ .Description }}
 {{ end }}
+## Getting Started
+This project uses [devbox](https://github.com/jetpack-io/devbox) to manage its development environment.
+
+Install devbox:
+```sh
+curl -fsSL https://get.jetpack.io/devbox | bash
+```
+
+Start the devbox shell:
+```sh 
+devbox shell
+```
```

Comment:
WDYT about nudging people towards using direnv as well? It tends to be significantly faster than starting a `devbox shell` (since we don't start a new shell process)

Reply:
Feels weird to me since I personally don't use it (I tend to use `devbox run` a lot which is faster than shell).

Feels a bit prescriptive no? i.e. devbox is absolutely needed to develop on this repo, but direnv is more a matter of opinion? 

Anyway, a follow up to this could be to allow people to specify their own readme templates so they can modify as needed.

Reply:
its just sooo much faster